docs/testing/testing.md

```diff
@@ -125,18 +125,6 @@ something valuable to helping keep Zulip bug-free.
     git remote add -f upstream https://github.com/zulip/zulip.git
   ```
 
-- When building the development environment using Vagrant and the LXC
-  provider, if you encounter permissions errors, you may need to
-  `chown -R 1000:$(id -g) /path/to/zulip` on the host before running
-  `vagrant up` in order to ensure that the synced directory has the
-  correct owner during provision. This issue will arise if you run `id
-  username` on the host where `username` is the user running Vagrant
-  and the output is anything but 1000.
-  This seems to be caused by Vagrant behavior; for more information,
-  see [the vagrant-lxc FAQ entry about shared folder permissions][lxc-sf].
-
-[lxc-sf]: https://github.com/fgrehm/vagrant-lxc/wiki/FAQ#help-my-shared-folders-have-the-wrong-owner
```

Comment:
How do user IDs get handled using `vagrant-docker` with a host user ID that's not 1000?  I'm curious if there's a similar problem.

Reply:
Yeah, there is a similar problem. I wonder if we can fix it by changing the `vagrant` uid in the `Dockerfile`.

Reply:
Fixed in #12418.